### DISCUSSION TITLE
Control of Corsair PSU fan via iCUE

### DISCUSSION THREAD

**Discussion by FiRem00** [Rem0o/FanControl.Releases#2267] [Ideas]
Hi, would there be an option to control the PSU fan of for example, an AX1600i? 
![image](https://github.com/Rem0o/FanControl.Releases/assets/22938491/78139aaa-88a6-459f-84cb-59b6104202aa)

**Comment by Rem0o** [comment]
https://github.com/EvanMulawski/FanControl.CorsairLink
